### PR TITLE
Add current lambda functions to repository to track them.

### LINKED ISSUE #190: Add redirects using AWS Lambda
Following files need to be redirects using serverless functions in AWS Lambda:

```
source/migrating-v1-to-v2.rst => source/payments/migrating-v1-to-v2.rst
source/guides/multicurrency.rst => source/payments/multicurrency.rst
source/guides/recurring.rst => source/payments/recurring.rst
source/guides/payment-status-changes.rst => source/payments/status-changes.rst (folder + filename aangepast)
source/payments/webhooks.rst => source/guides/webhooks.rst
source/security.rst  => source/guides/security.rst
```

While we're at it, we should probably also redirect top level routes:
```
reference/v2 => reference/v2/payments-api/create-payment
reference/v1 => reference/v1/payments-api/create-payment
```

### PR DESCRIPTION
**Pull request by hvalle**
These are the current lambda functions used. Added them to give visibility and track them.

Fixes #190.

### CODE REVIEW

File: .lambda-functions/origin-response.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    headers['x-frame-options'] = [{ key: 'X-Frame-Options', value: 'DENY' }];
+    headers['x-xss-protection'] = [{ key: 'X-XSS-Protection', value: '1; mode=block' }];
+    headers['referrer-policy'] = [{ key: 'Referrer-Policy', value: 'same-origin' }];
+    headers['expect-ct'] = [{ key: 'Expect-CT', value: 'max-age=0, report-uri="https://mollie.report-uri.com/r/d/ct/reportOnly"' }];
+
+    delete headers["server"];
+
+    // Configure the URL redirects
+    switch (request.uri) {
+        case '/migrating-v1-to-v2':
```
Comment (vernondegoede) [inline .lambda-functions/origin-response.js]: Can we use Regex here? `/migrating-v1-to-v2` works fine, but `/migrating-v1-to-v2/` doesn't.

File: .lambda-functions/origin-response.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+            break;
+        case '/reference/v2':
+            redirectTo('/reference/v2/payments-api/create-payment');
+            break;
+        case '/reference/v1':
+            redirectTo('/reference/v1/payments-api/create-payment');
+            break;
+    }
+
+    //Return modified response
```
Comment (vernondegoede) [inline .lambda-functions/origin-response.js]: Nitpick: missing space between `//` and `Return`.

https://github.com/airbnb/javascript#comments--spaces

File: .lambda-functions/origin-response.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
+'use strict';
+exports.handler = (event, context, callback) => {
+
+    const response = event.Records[0].cf.response;
+    const request = event.Records[0].cf.request;
+    const headers = response.headers;
```
Comment (vernondegoede) [inline .lambda-functions/origin-response.js]: I'd prefer to use `response.headers` everywhere instead of passing this by reference.

**Comment by willemstuursma** [comment]
It would be great if there could be a mention of the lambdas to show of that we are using serverless for this.

File: .lambda-functions/origin-response.js (hunk 1 above)
Comment (hvalle) [reply .lambda-functions/origin-response.js]: Regex is not fully supported on switch statements, but I could do a simple `switch (request.uri.replace(/\/$/, ''))` to remove the trailing `/` when matching.

**Comment by hvalle** [comment]
@willemstuursma you mean in the main readme of this repository?

**Comment by vernondegoede** [comment]
@martijnpolder Can this one be merged?

**Pull request merged**